doc/source/whatsnew/v1.3.0.rst

```diff
@@ -974,6 +974,7 @@ Other
 - Bug in :meth:`DataFrame.convert_dtypes` incorrectly raised ValueError when called on an empty DataFrame (:issue:`40393`)
 - Bug in :meth:`DataFrame.clip` not interpreting missing values as no threshold (:issue:`40420`)
 - Bug in :class:`Series` backed by :class:`DatetimeArray` or :class:`TimedeltaArray` sometimes failing to set the array's ``freq`` to ``None`` (:issue:`41425`)
+- Bug in :meth:`CustomBusinessMonthBegin` (:meth:`CustomBusinessMonthEnd`) not applying the extra ``offset`` parameter when beginning (end) of the target month is already a business day (:issue:`41356`)
```

Comment:
you've given the class but not the method.  Is it e.g. `CustomBusinessMonthBegin.__add__`?

Reply:
Oops, my apologies. It should be corrected now.